examples/consumer/main.go

```diff
@@ -335,6 +335,7 @@ func initSubscribers(cType ConsumerTypeEnum) map[string]string {
 		subscribeTo[string(ptpEvent.PtpClockClassChange)] = string(ptpEvent.PtpClockClass)
 		subscribeTo[string(ptpEvent.PtpStateChange)] = string(ptpEvent.PtpLockState)
 		subscribeTo[string(ptpEvent.GnssStateChange)] = string(ptpEvent.GnssSyncStatus)
+		subscribeTo[string(ptpEvent.SyncStateChange)] = string(ptpEvent.SyncStatusState)
```

Comment:
Consumer  don't subscribe to this state , I think this they get by default . when they subscribe to any other event types